### PR TITLE
jupyter compatible keybindings

### PR DESCRIPTION
**Pull request by pfitzseb**
This switches our code-eval keybindings to something close to Jupyter and makes sure our kbds aren't used in jupyter notebooks.

### CODE REVIEW

**Review by aviatesk — APPROVED** [review]
lgtm, I would like to wait for approve from @davidanthoff before merging tho

**Comment by aviatesk** [comment]
Could you take a look at this, @davidanthoff ?
Well I think we can move forward with this anyway; afaiu the keybindings are just compatible with the Jupyter extension's.

**Comment by davidanthoff** [comment]
I would do this once the notebook stuff in base VS Code has shipped and we are sure that we know what their final choices will be.

**Comment by davidanthoff** [comment]
Or wait, is this not actually updating our keybindings, but just making sure that they are not used in the notebook UI? In that case, I'm in favor, of course.

**Comment by pfitzseb** [comment]
I was sneaky -- it does both. I actually thought the November release would finalize the Notebook API, but it turns out it didn't. It does seem *very* unlikely that the notebook keybindings will change though.

**Pull request closed without merging**
